etcd/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "0.1.0",
+  "max_agent_version": "6.0.0",
+  "min_agent_version": "5.6.3",
+  "name": "etcd",
+  "short_description": "etcd description.",
+  "support": "contrib",
```

Comment:
Yeah they should be core. The rake task creates it like this. We'll have to update a bunch of these to make them better. For now, all that really matters is the version. But the other stuff should be fixed too